atomicfu/build.gradle

```diff
@@ -1,386 +0,0 @@
-            }
-        }
-
-        wasmWasiMain {
-            dependsOn(sourceSets.jsAndWasmSharedMain)
-            dependencies {
-                compileOnly 'org.jetbrains.kotlin:kotlin-stdlib-wasm-wasi'
-            }
-        }
-        wasmWasiTest {
-            dependencies {
-                implementation 'org.jetbrains.kotlin:kotlin-test-wasm-wasi'
-            }
-        }
-
-        jvmMain {
-            dependencies {
-                compileOnly 'org.jetbrains.kotlin:kotlin-stdlib'
-            }
-        }
-        jvmTest {
-            dependencies {
-                implementation 'org.jetbrains.kotlin:kotlin-reflect'
-                implementation 'org.jetbrains.kotlin:kotlin-test'
-                implementation 'org.jetbrains.kotlin:kotlin-test-junit'
-                implementation libs.junit.junit
-            }
-        }
-    }
-}
-
-// Support of all non-deprecated targets from official tier list: https://kotlinlang.org/docs/native-target-support.html
-kotlin {
-    // Tier 1
-    macosX64()
-    macosArm64()
-    iosSimulatorArm64()
-    iosX64()
-
-    // Tier 2
-    linuxX64()
-    linuxArm64()
-    watchosSimulatorArm64()
-    watchosX64()
-    watchosArm32()
-    watchosArm64()
-    tvosSimulatorArm64()
-    tvosX64()
-    tvosArm64()
-    iosArm64()
-
-    // Tier 3
-    androidNativeArm32()
-    androidNativeArm64()
-    androidNativeX86()
-    androidNativeX64()
-    mingwX64()
-    watchosDeviceArm64()
-
-    applyDefaultHierarchyTemplate { target ->
-        target.group("native") { nativeTargets ->
-            nativeTargets.group("nativeUnixLike") {
-                it.withLinux()
-                it.withApple()
-            }
-            nativeTargets.group("androidNative32Bit") {
-                it.withAndroidNativeX86()
-                // This is a WA: in KotlinHierarchyBuilderImpl.withAndroidNativeArm32 the KonanTarget equals ANDROID_X86 instead of ANDROID_ARM32
-                it.withCompilations { compilation ->
-                    compilation.target instanceof KotlinNativeTarget && (compilation.target as KotlinNativeTarget).konanTarget.name == "android_arm32"
-                }
-            }
-            nativeTargets.group("androidNative64Bit") {
-                it.withAndroidNativeArm64()
-                it.withAndroidNativeX64()
-            }
-        }
-    }
-
-    sourceSets {
-        nativeMain { dependsOn commonMain }
-
-        nativeTest {}
-
-        nativeUnixLikeMain {
-            kotlin.srcDir("src/nativeUnixLikeMain/kotlin")
-            dependsOn nativeMain
-        }
-
-        androidNative32BitMain {
-            kotlin.srcDir("src/androidNative32BitMain/kotlin")
-            dependsOn nativeMain
-        }
-
-        androidNative64BitMain {
-            kotlin.srcDir("src/androidNative64BitMain/kotlin")
-            dependsOn nativeMain
-        }
-
-        mingwX64Main { dependsOn nativeMain }
```

Comment:
Removed this dependsOn, because we have such hierarchy `mingwX64Main` -> `mingwMain` -> `nativeMain`, thus we don't need an additional edge `mingwX64Main` -> `nativeMain`